tests/test_core/test_layer_decay_optimizer_constructor.py

```diff
@@ -1,11 +1,167 @@
 # Copyright (c) OpenMMLab. All rights reserved.
 import torch
 import torch.nn as nn
+from mmcv.cnn import ConvModule
 
-from mmseg.core.layer_decay_optimizer_constructor import \
-    LayerDecayOptimizerConstructor
+from mmseg.core.optimizers.layer_decay_optimizer_constructor import (
+    LayerDecayOptimizerConstructor, LearningRateDecayOptimizerConstructor)
```

Comment:
Might test `layer_wise_vit` instead of `LayerDecayOptimizerConstructor ` which is the derecated one

Reply:
we need to modify it again after deleting it